controllers/cluster_controller_phases_test.go

```diff
@@ -22,16 +22,16 @@ import (
 	"time"
 
 	. "github.com/onsi/gomega"
-	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/client-go/kubernetes/scheme"
-	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
-	capierrors "sigs.k8s.io/cluster-api/errors"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+
```

Comment:
Nit: Extra space

Reply:
I've been seeing a lot of these lately, while it's nice that they are a group, it requires manual intervention to keep it up-to-date, and I'd prefer to just use `goimports` instead

Reply:
These are sorted with `goimports`, and is the automated reformatting applied by GoLand.

Reply:
If you remove the space and run `goimports -w ...` on it, it should reorder them properly, I was mostly pointing out that the extra space/group needs to be maintained

Reply:
One could also run `goimports -w -local=sigs.k8s.io/cluster-api` and it would automatically order it this way.